php/libraries/Recording.class.inc

```diff
@@ -134,14 +134,18 @@ class Recording
                        SELECT
                          Value
                        FROM
-                         physiological_parameter_file pf 
-                       JOIN parameter_type pt
-                         USING (ParameterTypeID)
-                       JOIN physiological_file f
-                         USING (PhysiologicalFileID)
+                         physiological_parameter_file
                        WHERE
-                         f.PhysiologicalFileID = :v_fileid AND
-                         pt.Name = :v_headername
+                         ParameterTypeID = (
+                           SELECT ParameterTypeID 
+                           FROM parameter_type
+                           WHERE Name = :v_headername
+                         )
+                         AND PhysiologicalFileID = (
+                           SELECT PhysiologicalFileID 
+                           FROM physiological_file
+                           WHERE PhysiologicalFileID = :v_fileid
+                         )
```

Comment:
@cmadjar Are you sure it is necessary to include this part of the query?

This could be resumed to:
`AND PhysiologicalFileID = :v_fileid`
or there is a bug in the logic of this query.
Selecting the same field that was use in the where clause equal the value used in the where clause. Unless there is no row with this value, in that case, 'NULL' will be returned by the nested query and nothing is equal to NULL so the outside query will not match any row.

Reply:
hahahaha, I can't believe I wrote that... My sick lung was starting to affect my brain I guess ;)

Reply:
PS: fixed in last commit